src/database/migrations/20190905181413-create-like.js

```diff
@@ -0,0 +1,38 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Likes', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    likedBy: {
+      type: Sequelize.INTEGER,
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+      references: {
+        model: 'Users',
+        key: 'id'
+      }
+    },
+    accommodationId: {
+      type: Sequelize.INTEGER,
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+      references: {
+        model: 'Accommodation',
+        key: 'id'
+      }
```

Comment:
This is a self relationship right? Why do you have this?

Reply:
Nope. It associates a like to an accommodation. This field is necessary in the migration so association can be possible in the models.